index.js

```diff
@@ -9,6 +9,8 @@
 var loopback = require('loopback');
 var PushConnector = require('./lib/push-connector');
 exports = module.exports = PushConnector;
+var SG = require('strong-globalize');
```

Comment:
Why not following strong-globalize coding guideline here?
https://www.npmjs.com/package/strong-globalize#autonomous-message-loading
Please study paragraphs after "For example, the following does not work as intended because the package sub calls SG.SetRootDir first:"